ngraph/test/onnx/onnx_import_library.cpp

```diff
@@ -25,16 +23,6 @@ using namespace ngraph;
 
 static std::string s_manifest = "${MANIFEST}";
 
-NGRAPH_TEST(onnx, get_function_op_with_version)
```

Comment:
Why did you remove this test? Don't we need it anymore?

Reply:
I've removed the same test in https://github.com/openvinotoolkit/openvino/pull/4107 because it was causing locally unreproducible failures on Ubuntu 18 and Ubuntu 20 in CI. This test just checks if the ONNX library providing schema access is linked but now we have a better test that checks if the ONNX schema is available and lets us expand ONNX functions to operators https://github.com/openvinotoolkit/openvino/blob/master/ngraph/test/onnx/onnx_import.in.cpp#L470 + https://github.com/openvinotoolkit/openvino/blob/master/ngraph/frontend/onnx_import/src/core/transform.cpp#L44